tests/unit/test_artifactory_path.py

```diff
@@ -967,5 +971,190 @@ def test_from_aql_file(self):
         assert artifact.root == "/reponame/"
 
 
+class TestArtifactoryPathGetAll(unittest.TestCase):
+    # TODO: test repositories and permissions
+    def setUp(self):
+        self.arti = ArtifactoryPath("http://b.com/artifactory")
+
+    def test_get_all_users(self):
+        url = "/api/security/users"
```

Comment:
Can we move the preparation to a fixture? It's hard to get "what we actually do and what asserts are about" from long tests

Reply:
All fixtures are defined in `setUp`.